package.json

```diff
@@ -4,7 +4,10 @@
   "description": "Compiled static assets for Water Data For The Nation",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "npm run test:karma",
+    "test:jasmine": "jasmine assets/tests/**",
```

Comment:
I'm not familiar with running tests using just a jasmine command. What is the difference between this on using karma?

Reply:
That will run via node.js rather than in a browser. I suppose it remains to be seen what our tests will look like and whether that will be useful, but I thought I'd just put it in there for now.

Reply:
That's pretty cool. :thinking:

Reply:
Since our javascript code will always run in a browser we should be using the test:karma. I lean towards removing it unless we have javascript that is not run in a browser to test.

Reply:
Ok, I'll remove it.